README.md

```diff
@@ -1,2 +1,36 @@
-# pgbouncer-container
-A docker container, containing the pgbouncer connection pooler. Built for use in the pgbouncer k8s charm: https://github.com/canonical/pgbouncer-k8s-operator
+# PgBouncer Container
+
+This docker image defines and configures the [PgBouncer connection pooler for PostgreSQL](https://www.pgbouncer.org), with a set of sensible defaults in pgbouncer.ini. These defaults are insecure, and should be modified to suit your needs, specifically in pgbouncer.ini and userlist.txt.
+
+Built for use in the pgbouncer k8s charm: https://github.com/canonical/pgbouncer-k8s-operator
+
+## Usage, and connecting to a postgres container
+
+1 in one terminal, run the following:
+  1 `docker pull postgres/postgres`
```

Comment:
Should it be `docker pull postgres`, `docker pull ubuntu/postgres` or something else? Checking here, I could not find this image.

Reply:
You are correct, I've changed it to `docker pull postgres`. These instructions are generally likely to change once I've got a